lib/styles/settings/_typography.scss

```diff
@@ -1,176 +1,199 @@
-$font-size-info-s: 8px;
-$line-height-info-s: 12px;
+$font-size-info-s-DEPRECATED: 8px;
+$line-height-info-s-DEPRECATED: 12px;
 
-$font-size-info-m: 10px;
-$line-height-info-m: 12px;
+$font-size-info-m-DEPRECATED: 10px;
+$line-height-info-m-DEPRECATED: 12px;
 
-$font-size-text-xs: 10px;
-$line-height-text-xs: 14px;
+$font-size-text-xs-DEPRECATED: 10px;
+$line-height-text-xs-DEPRECATED: 14px;
 
-$font-size-text-s: 12px;
-$line-height-text-s: 18px;
+$font-size-text-s-DEPRECATED: 12px;
+$line-height-text-s-DEPRECATED: 18px;
 
-$font-size-text-m: 14px;
-$line-height-text-m: 20px;
+$font-size-text-m-DEPRECATED: 14px;
+$line-height-text-m-DEPRECATED: 20px;
 
-$font-size-text-l: 16px;
-$line-height-text-l: 24px;
+$font-size-text-l-DEPRECATED: 16px;
+$line-height-text-l-DEPRECATED: 24px;
 
-$font-size-text-xl: 18px;
-$line-height-text-xl: 26px;
+$font-size-text-xl-DEPRECATED: 18px;
+$line-height-text-xl-DEPRECATED: 26px;
 
-$font-size-headline-xxs: 12px;
-$line-height-headline-xxs: 16px;
+$font-size-headline-xxs-DEPRECATED: 12px;
+$line-height-headline-xxs-DEPRECATED: 16px;
 
-$font-size-headline-xs: 14px;
-$line-height-headline-xs: 16px;
+$font-size-headline-xs-DEPRECATED: 14px;
+$line-height-headline-xs-DEPRECATED: 16px;
 
-$font-size-headline-s: 16px;
-$line-height-headline-s: 20px;
+$font-size-headline-s-DEPRECATED: 16px;
+$line-height-headline-s-DEPRECATED: 20px;
 
-$font-size-headline-m: 20px;
-$line-height-headline-m: 28px;
+$font-size-headline-m-DEPRECATED: 20px;
+$line-height-headline-m-DEPRECATED: 28px;
 
-$font-size-headline-l: 24px;
-$line-height-headline-l: 32px;
+$font-size-headline-l-DEPRECATED: 24px;
+$line-height-headline-l-DEPRECATED: 32px;
 
-$font-size-headline-xl: 28px;
-$line-height-headline-xl: 38px;
+$font-size-headline-xl-DEPRECATED: 28px;
+$line-height-headline-xl-DEPRECATED: 38px;
 
-$font-size-title-s: 36px;
-$line-height-title-s: 52px;
+$font-size-title-s-DEPRECATED: 36px;
+$line-height-title-s-DEPRECATED: 52px;
 
-$font-size-title-m: 44px;
-$line-height-title-m: 58px;
+$font-size-title-m-DEPRECATED: 44px;
+$line-height-title-m-DEPRECATED: 58px;
 
-$font-size-title-l: 52px;
-$line-height-title-l: 68px;
+$font-size-title-l-DEPRECATED: 52px;
+$line-height-title-l-DEPRECATED: 68px;
 
-$font-size-button-xs: 8px;
-$line-height-button-xs: 10px;
+$font-size-button-xs-DEPRECATED: 8px;
+$line-height-button-xs-DEPRECATED: 10px;
 
-$font-size-button-s: 10px;
-$line-height-button-s: 12px;
+$font-size-button-s-DEPRECATED: 10px;
+$line-height-button-s-DEPRECATED: 12px;
 
-$font-size-button-m: 12px;
-$line-height-button-m: 16px;
+$font-size-button-m-DEPRECATED: 12px;
+$line-height-button-m-DEPRECATED: 16px;
 
-$font-size-button-l: 14px;
-$line-height-button-l: 16px;
+$font-size-button-l-DEPRECATED: 14px;
+$line-height-button-l-DEPRECATED: 16px;
 
-@mixin textInfoS {
-	font-size: $font-size-info-s;
-	line-height: $line-height-info-s;
+@mixin textInfoS-DEPRECATED {
+	@warn "This mixin is deprecated!";
```

Comment:
Did you check how many worning will it generate on platform? Is it still posibble to read Vite output/stacktrace between these warnings?

Reply:
Right, it's redundant.